provider/test-programs/cloudfunctions-function/Pulumi.yaml

```diff
@@ -0,0 +1,47 @@
+name: test
+resources:
+    archive:
+        options:
+            provider: ${provider}
+        properties:
+            bucket: ${bucket.name}
+            source:
+                fn::fileAsset: ./src/src.zip
+        type: gcp:storage:BucketObject
+    bucket:
+        options:
+            provider: ${provider}
+        properties:
+            location: US
+        type: gcp:storage:Bucket
+    # invoker:
+    #     options:
+    #         provider: ${provider}
+    #     properties:
+    #         cloudFunction: ${tested-resource.name}
+    #         member: allUsers
+    #         # project: ${tested-resource.project}
+    #         # region: ${tested-resource.region}
+    #         role: roles/cloudfunctions.invoker
+    #     type: gcp:cloudfunctions:FunctionIamMember
```

Comment:
I assume this was part of the hand editing to make this one run, maybe add a comment for context or remove?